### PR TITLE
Bigtable to vector embeddings

### PR DESCRIPTION
**Pull request by ron-gal**

### CODE REVIEW

File: v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java [hunk 1]
```diff
@@ -0,0 +1,630 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Dataflow pipeline that exports data from a Cloud Bigtable table to JSON files in GCS,
+ * specifically for Vector Embedding purposes. Currently, filtering on Cloud Bigtable table is not
+ * supported.
+ *
+ * <p>Check out <a href=
+ * "https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v1/README_Cloud_Bigtable_to_GCS_JSON.md">README</a>
```
Comment (bvolpato) [inline v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java:61]: nit `Cloud_Bigtable_to_Vector_Embeddings`

File: v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+    private VectorEmbeddings vectorEmbeddings;
+
+    public BigtableToVectorEmbeddingsFn(
+        ValueProvider<String> columnsAliasesProvider,
+        ValueProvider<Integer> embeddingByteSizeProvider,
+        ValueProvider<String> allowRestrictsProvider,
+        ValueProvider<String> denyRestrictsProvider,
+        ValueProvider<String> intNumericRestrictsProvider,
+        ValueProvider<String> floatNumericRestrictsProvider,
+        ValueProvider<String> doubleNumericRestrictsProvider) {
```
Comment (bvolpato) [inline v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java:320]: nit / style: a builder pattern here could prevent from messing up parameter orders

**Review by bvolpato — APPROVED** [review]
LGTM! Thanks. Just added 2 nits but no blockers

**Review by bvolpato — CHANGES_REQUESTED** [review]
In fact, can you please rebase / squash this into one or two commits? When looking to merge, we'd bring 38 commits to main history

File: v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java (hunk 1 above)
Comment (ron-gal) [reply v1/src/main/java/com/google/cloud/teleport/bigtable/BigtableToVectorEmbeddings.java]: Done

**Review by bvolpato — DISMISSED** [review]
LGTM, thanks!

**Pull request merged**